nsis/install.nsi

```diff
@@ -311,6 +314,25 @@ Section $(TITLE_SEC_ENV) SEC07
   SendMessage ${HWND_BROADCAST} ${WM_WININICHANGE} 0 "STR:Environment" /TIMEOUT=5000
 SectionEnd
 
+!macro WRITE_FILE_ASSOC_KEYS EXT EXECUTABLE DESCRIPTION ICON_NAME
+  WriteRegExpandStr ${SOFTWARE_CLASSES_ROOT_KEY} "Software\Classes\.${EXT}\OpenWithProgids\" "${REG_VALUE_NAME}.${EXT}" ""
+  
+  WriteRegExpandStr ${SOFTWARE_CLASSES_ROOT_KEY} "Software\Classes\${REG_VALUE_NAME}.${EXT}" "" "${DESCRIPTION}"
+  
+  WriteRegExpandStr ${SOFTWARE_CLASSES_ROOT_KEY} "Software\Classes\${REG_VALUE_NAME}.${EXT}\" "DefaultIcon" "$INSTDIR\share\kicad\${ICON_NAME}"
+  WriteRegExpandStr ${SOFTWARE_CLASSES_ROOT_KEY} "Software\Classes\${REG_VALUE_NAME}.${EXT}\shell\open\command" "" "$INSTDIR\bin\${EXECUTABLE}"
+!macroend
+
+Section $(TITLE_FILE_ASSOC) SEC08
```

Comment:
Please add MUI_DESCRIPTION_TEXT for SEC08 as is done in MUI_FUNCTION_DESCRIPTION_BEGIN